lib/rbac/seed.rb

```diff
@@ -123,11 +123,15 @@ def find_uuid(type, data, name)
       result.uuid
     end
 
-    def create_request(user_file)
-      raise "File #{user_file} not found" unless File.exist?(user_file)
+    def create_or_use_request(user_file = nil)
+      if user_file.nil?
+        ManageIQ::API::Common::Request.current
```

Comment:
@syncrou should we raise an error if the ManageIQ::API::Common::Request.current is empty?

Also this seems a bit confusing can we do something like this

@request = ManageIQ::API::Common::Request.current || make_request(user_file)

Reply:
Yeah, I was confused on the best behavior here too. I was trying to keep all the behavior intact when the seeding file is used in a rake task, but thinking through this, after adding this endpoint there probably is no reason that the rake task will be used much if at all.